src/ui/camera.js

```diff
@@ -829,116 +870,16 @@ class Camera extends Evented {
         if (tr.mercatorFromTransition && zoom < halfZoomTransition) {
             tr.setProjection({name: 'globe'});
             tr.zoom = zoom;
-            return this._cameraForBoxAndBearing(tr, p0, p1, bearing, options);
+            return this._cameraForAabb(tr, p0, p1, bearing, options);
```

Comment:
I think this name has the potential to be confused with a function taking input as 3d AABB. Maybe consider renaming it `_cameraForBounds` or `_cameraForBoundsAndBearing`?